nvflare/apis/job_def.py

```diff
@@ -144,16 +144,18 @@ def job_from_meta(meta: dict) -> Job:
     """Converts information in meta into a Job object.
 
     Args:
-        meta: dict of meta information
+        meta: dict of meta information(
 
     Returns:
         A Job object.
     """
     job = Job(
-        meta.get(JobMetaKey.JOB_ID),
-        meta.get(JobMetaKey.STUDY_NAME),
-        meta.get(JobMetaKey.RESOURCE_SPEC),
-        meta.get(JobMetaKey.DEPLOY_MAP),
-        meta,
+        job_id=meta.get(JobMetaKey.JOB_ID),
+        study_name=meta.get(JobMetaKey.STUDY_NAME),
+        resource_spec=meta.get(JobMetaKey.RESOURCE_SPEC),
+        deploy_map=meta.get(JobMetaKey.DEPLOY_MAP),
+        meta=meta,
+        min_sites=meta.get(JobMetaKey.MIN_CLIENTS),
```

Comment:
If JobMetaKey.MIN_CLIENTS in missing, then this going to set min_sites = None.

We should change this line to `meta.get(JobMetaKey.MIN_CLIENTS, 1)`

Reply:
fixed.